### PR TITLE
[Minor] Advanced tokens for Checkbox component

### PR DESCRIPTION
**Pull request by jsuarezgonz**

### CODE REVIEW

File: lib/src/common/variables.js [hunk 1]
```diff
@@ -252,15 +252,27 @@ export const componentTokens = {
...
+    disabledBorderColor:globalTokens.softBlue,
+    disabledBorderColorOnDark:globalTokens.softBlue,
+    disabledColorOnDark: "#BFBFBF",
     disabledCheckColor: globalTokens.white,
+    disabledCheckColorOnDark: "#FFFFFF",
     disabledFontColor: globalTokens.lighterBlack,
+    disabledFontColorOnDark: "#575757",
     focusColor: globalTokens.blue,
-    fontFamily: globalTokens.type_sans,
+    checkGutter: "10px"
```
Comment (aweell) [inline lib/src/common/variables.js:275]: I suggest changing `checkGutter` to `checkLabelSpacing` since is more semantic

**Review by aweell — COMMENTED** [review]
Should we add tokens for the hover state? In order to provide a cue, like changing the bg or border color on hover?

**Review by aweell — CHANGES_REQUESTED** [review]
I saw you added `backgroundHoverColorChecked`, should we have one for unchecked also?

**Comment by jsuarezgonz** [comment]
> I saw you added `backgroundHoverColorChecked`, should we have one for unchecked also?

There is no token referring to the 'unchecked' component, we only have the border in that case; that´s why I created a hover token for the border and not for the background.

**Review by aweell — APPROVED** [review]
🤘🏼

**Pull request merged**